bin/pa11y-ci.js

```diff
@@ -227,6 +228,15 @@ function loadSitemapIntoConfig(program, config) {
 				xmlMode: true
 			});
 
+			const isSitemapIndex = $('sitemapindex').length > 0;
+			if (isSitemapIndex) {
+				return Promise.all($('sitemap > loc').toArray().map(element => {
+					return getUrlsFromSitemap($(element).text(), config);
+				})).then(configs => {
+					return configs.pop();
```

Comment:
If i'm reading this correctly, the code is going to load every sitemap in a `<loc>` but then only return the last configuration.
It seems sensible to either support loading all sitemaps from an index file and merging them together into one config. Or not add this complexity at all, as currently this code is synonymous to just loading the last sidemap directly.